WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -0,0 +1,77 @@
+package org.wordpress.android.ui.people.utils;
+
+import com.android.volley.VolleyError;
+import com.wordpress.rest.RestRequest;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.wordpress.android.WordPress;
+import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.AppLog.T;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PeopleUtils {
+
+    public static void fetchUsers(String siteID, final int localTableBlogId, final PeopleUtils.Callback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null) {
+                    try {
+                        JSONArray jsonArray = jsonObject.getJSONArray("users");
+                        List<Person> people = peopleListFromJSON(jsonArray, localTableBlogId);
+                        PeopleTable.savePeople(people);
```

Comment:
I've tried adding a different method which calls the `fetchUsers` but that introduces a lot of unnecessary complications/callbacks. I don't think it's worth it to add +40 lines just for this. So, I opted to go with your initial suggestion of moving the save to the callback in the Activity.

For the sake of transparency, I don't like this solution at all. I don't like moving something like a DB save to Activity just to make a method unit testable. In any case, this was the simplest solution to the issue you brought up, so I guess it's fine to go with it. Once wp-stores are ready, we'll be changing these to the recommended structure anyway. Thanks!